organize/models.py

```diff
@@ -30,7 +31,7 @@ class EventApplication(models.Model):
     city = models.CharField(max_length=200)
     country = models.CharField(max_length=200, choices=countries)
     latlng = models.CharField(max_length=30, null=True, blank=True)
-    website_slug = models.SlugField()
+    website_slug = AutoSlugField(populate_from='city', unique=True, editable=True)
```

Comment:
This shouldn't be unique -- there are gonna be multiple applications that can have the same slug.